### ISSUE TITLE
Change (remove?) the capitalization of the experiments displayed in the right side panel

### ISSUE THREAD

**Issue by jypeter** [aims-group/publication-site#203]
When I look at the experiments (https://cmip-publications.llnl.gov/?type=experiment), the experiments names listed in the panel on the right side always start with a capital letter

- `lgm `=> `Lgm`
- `piControl `=> `PiControl`
- `midHolocene `=> `MidHolocene`
- And So On

This is not a major problem and is probably used for sorting the displayed information, but it is slightly confusing

**Comment by durack1** [comment]
@jypeter good point, these should mirror the experiment names listed in https://github.com/WCRP-CMIP/CMIP6_CVs/blob/main/CMIP6_experiment_id.json identically. Unfortunately, we don't have similar CVs defined for CMIP5, so this might have been part of the error source. @mauzey1 can you add this to the list? Hopefully it's a small task

**Comment by jypeter** [comment]
I have just noticed that the capitalization displayed at the top of the page seems correct when you actually select an experiment!

- [Lig127k](https://cmip-publications.llnl.gov/?type=experiment&option=lig127k) => `all: lig127k`
- [PiControl](https://cmip-publications.llnl.gov/?type=experiment&option=piControl) => `all: piControl`

There must be a file with a frozen list of CMIP5 experiment names, even if there was no official CV page
